src/app/shared/image/image-upload.component.ts

```diff
@@ -25,6 +26,9 @@ export class ImageUploadComponent {
   @Input() minWidth = 200;
   @Input() minHeight = 200;
 
+  private imgError: string;
```

Comment:
I think the linter will eventually yell at us for having private variables that are used in the template -- @cavis changed a lot of them in #128